commit 0f64ac7c10fb4053f578ef950e55510818d6d440
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1334285499 -0500

    Remove version.h. Cruft from an older version prior to 0.9.6.
    Part of an extensive cleanup of various problems with kipi-plugins, digikam,
    and gwenview to resolve bug reports 241, 962, 963.

diff --git a/digikam/digikam/version.h b/digikam/digikam/version.h
deleted file mode 100644
index d9baae1..0000000
--- a/digikam/digikam/version.h
+++ /dev/null
@@ -1,29 +0,0 @@
-/* ============================================================
- *
- * This file is a part of digiKam project
- * http://www.digikam.org
- *
- * Date        : 2004-09-09
- * Description : digiKam release ID header.
- *
- * Copyright (C) 2004-2008 by Gilles Caulier <caulier dot gilles at gmail dot com>
- *
- * This program is free software; you can redistribute it
- * and/or modify it under the terms of the GNU General
- * Public License as published by the Free Software Foundation;
- * either version 2, or (at your option)
- * any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * ============================================================ */
-
-#ifndef DIGIKAM_VERSION_H
-#define DIGIKAM_VERSION_H
-
-static const char digikam_version[] = "0.9.4";
-
-#endif // DIGIKAM_VERSION_H
